internal/driver/glfw/window_test.go

```diff
@@ -258,6 +259,31 @@ func TestWindow_DragIntoNewObjectKeepingFocus(t *testing.T) {
 	assert.Nil(t, d2.popMouseEvent())
 }
 
+func TestWindow_NoDragEndWithoutDraggedEvent(t *testing.T) {
+	w := d.CreateWindow("Test").(*window)
+	w.Canvas().SetScale(1.0)
+	d := &draggableMouseableObject{Rectangle: canvas.NewRectangle(color.White)}
+	d.SetMinSize(fyne.NewSize(10, 10))
+	w.SetContent(d)
+
+	// wait for canvas to get its size right
```

Comment:
I removed all of the time code in window test and replaced it with two fixes (one for event queue wait and the other for a fore recalculate) - this makes the tests faster and more reliable.

Can the new pattern apply here using: `d.(*gLDriver).repaintWindow(w)` instead?

Reply:
Yes, of course. I'll catch up with the latest and greatest and then update the tests.